src/assets/chats/Ic_read.js

```diff
@@ -9,7 +8,7 @@ function Ic_read(props) {
         fillRule="evenodd"
         clipRule="evenodd"
         d="M15.452 1.377a.677.677 0 10-.969-.946L8.388 6.672l-.764-.902-.952.972 1.163 1.374a.677.677 0 001.002.035l6.615-6.774zm-4.064 0a.677.677 0 10-.97-.946L4.325 6.672 1.937 3.853a.677.677 0 10-1.034.876L3.77 8.116a.677.677 0 001.002.035l6.615-6.774z"
-        fill={props.fill ? props.fill : COLORS.blackgrey}
+        fill={props.fill ? props.fill : '#828282'}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The hardcoded color value `'#828282'` is used as a fallback if no `fill` prop is provided. This could lead to inconsistency in the app's color scheme if the color palette changes in the future. It would be better to import this color from a centralized color configuration file, ensuring consistency and ease of updates.

```diff
- fill={props.fill ? props.fill : '#828282'}
+ fill={props.fill ? props.fill : colors.blackgrey}
```
Please make sure to import the `colors` object at the top of your file.
```javascript
import { colors } from '../../utils/colors';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->